spec/support/shared_sites_controller_shared_behavior.rb

```diff
@@ -1,7 +1,15 @@
 shared_examples 'restricted to approved user' do |request_method, action, parameters = nil, sessions = nil, flash = nil|
```

Comment:
These specs are still generating deprecation warnings, because you are still passing the params & flash as a positional rather than a keyword argument. These can be re-written as:
```ruby
#note the omission of `sessions=nil`, which is not needed
shared_examples 'restricted to approved user' do |request_method, action, parameters = nil, flash = nil|

  context 'when user is not logged in' do
    it 'should redirect to login page' do
      send request_method, action, params: parameters, flash: flash
...etc.
```

Reply:
args is already passing parameters 
``` let(:args) do
    { params: parameters, flash: flash } ```
Reason it had to be set this way is because it gives an error if you pass session and just nil. you don't need it if its nil. But I figured some tests it may not be nil. So I check see if sessions exists before adding it to args.  I do not see positional error. Can you slack message me that? I will take a look.  If you have questions slack message me and we can discuss.

Reply:
Reason it had to be set this way is because it gives an error if you pass session and just nil. you don't need it if its nil. But I figured some tests it may not be nil. So I check see if sessions exists before adding it to args.  I do not see positional error. Can you slack message me that? I will take a look.  If you have questions slack message me and we can discuss.

Reply:
Sorry, I realized I was seeing those deprecation warnings on the main Rails5 branch, rather than this branch. However, I still prefer the style I suggested for a couple of reasons:
- `sessions` is never used, so we can remove that
- `send request_method, action, params: parameters, flash: flash` is more explicit and readable than `send request_method, action, args`

I'm also thinking we can omit the `flash`, which we never use in the specs